better_profanity/better_profanity.py

```diff
@@ -66,8 +66,12 @@ def add_censor_words(self, custom_words):
         self.CENSOR_WORDSET.update(custom_words)
 
     def contains_profanity(self, text):
-        """Return True if  the input text has any swear words."""
-        return text != self.censor(text)
+        """Return the first detected swear word if the input text has any swear words."""
```

Comment:
Change the name of this function to something that idiciates what the function does and returns